packages/core/schemas/pncQueryResult.ts

```diff
@@ -52,7 +52,7 @@ export const pncQueryResultSchema = z.object({
   forceStationCode: z.string().describe(ahoDescription.AnnotatedHearingOutcome.PncQuery.forceStationCode.$description),
   croNumber: z.string().optional(),
   checkName: z.string(),
-  pncId: z.string(),
+  pncId: z.string().optional(),
```

Comment:
Wouldn't mind seeing the example where this was not included

Reply:
Think this has been a misconception on my part. Will remove.